readme.md

```diff
@@ -188,6 +195,7 @@ Default actions:
 - `cut`
```

Comment:
You need to add it to the `The following options are ignored when `menu` is used:` list.